Xamarin.Essentials/Types/StorageExtensions.shared.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using Xamarin.Essentials;
+
+namespace Xamarin.Essentials.Types
+{
+    public static class StorageExtensions
```

Comment:
I am not sure if these _really_ add value... It is truly a single division operation.